0/release_version.py

```diff
@@ -166,6 +166,9 @@ def add_release(self, tag: str, branch: str, title: str, content: str, pre_relea
 
 
 def create_github_release():
+    if (only_author := os.getenv('PLUGIN_ONLY_COMMIT_AUTHOR')) and (only_author != os.getenv("DRONE_COMMIT_AUTHOR")):
+        return
```

Comment:
Maybe printing a warning or info message to let the user know that the release was skipped because of this condition would be handy?

Reply:
consider it done